resources/embedded/.editorconfig

```
@@ -0,0 +1,18 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 4
+indent_style = space
+insert_final_newline = true
+
+[*.yml]
+indent_size = 2
+
+[*.c, *.dts*, Makefile]
```

As I can't update the PR anymore, this needs to be squashed in:
```suggestion
[*.{c,dts*},Makefile*]
```